ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -66,566 +50,328 @@ const mountEngineGeneratePolicyToken = async (enginePath, secretPath, policy, ve
-      currentRouteName(),
-      'vault.cluster.secrets.backend.show',
-      'redirects to the show page'
-    );
-    assert.ok(showPage.editIsPresent, 'shows the edit button');
-    await deleteEngine(enginePath, assert);
-  });
-
-  test('it can create a secret with a non default max version and add metadata', async function (assert) {
-    assert.expect(4);
-    const enginePath = `kv-secret-${this.uid}`;
-    const secretPath = 'maxVersions';
-    const maxVersions = 101;
-    await mountSecrets.visit();
-    await mountSecrets.enable('kv', enginePath);
-    await settled();
-    await editPage.startCreateSecret();
-    await editPage.path(secretPath);
-    await editPage.toggleMetadata();
-    await settled();
-    await editPage.maxVersion(maxVersions);
-    await settled();
-    await editPage.save();
-    await settled();
-    await editPage.metadataTab();
-    await settled();
-    const savedMaxVersions = Number(
-      document.querySelector('[data-test-value-div="Maximum versions"]').innerText
-    );
-    assert.strictEqual(
-      maxVersions,
-      savedMaxVersions,
-      'max_version displays the saved number set when creating the secret'
-    );
-    // add metadata
-    await click('[data-test-add-custom-metadata]');
-    await fillIn('[data-test-kv-key]', 'key');
-    await fillIn('[data-test-kv-value]', 'value');
-    await click('[data-test-save-metadata]');
-    const key = document.querySelector('[data-test-row-label="key"]').innerText;
-    const value = document.querySelector('[data-test-row-value="key"]').innerText;
-    assert.strictEqual(key, 'key', 'metadata key displays after adding it.');
-    assert.strictEqual(value, 'value', 'metadata value displays after adding it.');
-    await deleteEngine(enginePath, assert);
-  });
-
-  skip('it can handle validation on custom metadata', async function (assert) {
-    assert.expect(3);
-    const enginePath = `kv-secret-${this.uid}`;
-    const secretPath = 'customMetadataValidations';
-
-    await mountSecrets.visit();
-    await mountSecrets.enable('kv', enginePath);
-    await settled();
-    await editPage.startCreateSecret();
-    await editPage.path(secretPath);
-    await editPage.toggleMetadata();
-    await settled();
-    await typeIn('[data-test-kv-value]', 'invalid\\/');
-    assert
-      .dom('[data-test-inline-error-message]')
-      .hasText('Custom values cannot contain a backward slash.', 'will not allow backward slash in value.');
-    await fillIn('[data-test-kv-value]', ''); // clear previous contents
-    await typeIn('[data-test-kv-value]', 'removed!');
-    assert.dom('[data-test-inline-error-message]').doesNotExist('inline error goes away');
-    await click('[data-test-secret-save]');
-    assert
-      .dom('[data-test-message-error]')
-      .includesText(
-        'custom_metadata validation failed: length of key',
-        'shows API error that is not captured by validation'
+  module('mount and configure', function () {
```

Comment:
To make clear which tests are for KV V1 vs V2 I separated into modules. The first module does not set up a secret engine on beforeEach